README.md

```diff
@@ -194,7 +194,7 @@ bat main.cpp | xclip
 `MANPAGER` environment variable:
 
 ```bash
-export MANPAGER="sh -c 'col -bx | bat -l man -p'"
+export MANPAGER="sh -c 'col -bx | bat -l man -p --paging=always --color=always | less -r'"
```

Comment:
Do we need both `--paging=always` and `less -r`?

Reply:
you are right, it does seems to work without `--paging=always`. 
Although, I do wonder if both of them are necessary when the set pager isn't less...